src/main/java/seedu/sudohr/model/events/Event.java

```diff
@@ -0,0 +1,93 @@
+package seedu.sudohr.model.events;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.sudohr.model.person.Person;
+
+
+/**
+ * Represents a Event in the sudohr book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Event {
+
+    private final Title title;
+    private final Set<Person> attendees;
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Event(Title title) {
+        this.title = title;
+        this.attendees = new HashSet<Person>();
```

Comment:
See if you can implement UniquePersonList to reuse the code for attendees, should be just a few changes in the class based on my experience with Department